tests/integration/like.test.js

```diff
@@ -0,0 +1,75 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import { startServer } from '../../src/server';
+
+const { expect } = chai;
+let app = null;
+let agent = null;
+let userToken;
+
+chai.use(chaiHttp);
+
+describe('LIKE ARTICLE', () => {
+  beforeEach(async () => {
+    app = await startServer(6600);
+    agent = chai.request(app);
+  });
+
+  describe('Patch /api/v1/articles/:articluleid/like', () => {
+    it('login User', (done) => {
+      agent
+        .post('/api/v1/login')
+        .send({ email: 'martins@gmail.com', password: '%RYYT&^UTB*UYT*IUYIU' })
```

Comment:
While this is good, you might want to add this to a mock file, import the file here and call the variable here.

Reply:
Also, note that your Travis is failing. You might want to review it

Reply:
@Raymond-Osy test is passing locally but it's failing on Travis I am still trying to figure out why